solution/ui/regulations/eregs-vite/src/views/Search.vue

```diff
@@ -240,10 +244,6 @@ export default {
     },
     data() {
         return {
-            base:
-                import.meta.env.VITE_ENV && import.meta.env.VITE_ENV !== "prod"
-                    ? `/${import.meta.env.VITE_ENV}`
-                    : "",
```

Comment:
I initially wrote this logic, and now I am crusading to expel it from the app.  

I've written another story to handle the API methods file, since it is beyond the scope of this bugfix story, and also will require refactoring the `fetchJson` api method that is used as the basis for all API calls from the Single Page Apps.  So, a bigger job that needs a dedicated tech debt story.